server/test/unit/server/webservices/views/test_util.py

```diff
@@ -54,3 +54,14 @@ def test_generate_json_response_with_pulp_encoder(self, mock_json):
         test_content = {'foo': 'bar'}
         util.generate_json_response_with_pulp_encoder(test_content)
         mock_json.dumps.assert_called_once_with(test_content, default=pulp_json_encoder)
+
+    def test_generate_response_redirect(self):
+        """
+        Test HttpResponseRedirect.
+        """
+        test_content = {'foo': 'bar'}
+        response = util.generate_json_response_with_pulp_encoder(test_content)
+        response = util.generate_redirect_response(response, '/some/url/')
```

Comment:
Generally I try to avoid the reassignment of a variable. At different times in the code `response` will take on different states which make it slightly more difficult to read.